vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -55,107 +26,40 @@ import dateFnsIsValid from 'date-fns/isValid';
           })
         );
 
-        const cleanString = tryCatchWrapper(function (string) {
-          // Clear any non ascii characters from the date string,
-          // mainly the LEFT-TO-RIGHT MARK.
-          // This is a problem for many Microsoft browsers where `toLocaleDateString`
-          // adds the LEFT-TO-RIGHT MARK see https://en.wikipedia.org/wiki/Left-to-right_mark
-          return string.replace(/[^\x00-\x7F]/g, '');
-        });
```

Comment:
This doesn't seem to be required anymore. It was added as part of https://github.com/vaadin/vaadin-date-picker-flow/pull/81 to fix an issue with Edge. Since then Edge has changed significantly to be based on Chrome, and the issue (using french locale) is not reproducable anymore in latest Edge - formatting and parsing return the same results as in Chrome.